src/main/java/net/rptools/maptool/client/functions/MacroFunctions.java

```diff
@@ -175,7 +175,7 @@ public Object getMacroButtonProps(Token token, int index, String delim) throws P
       } else {
         props.addProperty("tooltooltip", "");
```

Comment:
Looks like this will also create a bad property name "tooltooltip" in cases of missing tooltip.

Reply:
Good catch!